pkg/certs/cert-inspection/certgraphapi/type_registry.go

```diff
@@ -0,0 +1,98 @@
+package certgraphapi
+
+import "fmt"
+
+type PKIRegistryInfo struct {
```

Comment:
given a godoc comment indicating whether or not you anticipate this growing fields to represent information like, "supports offline-cert-regeneration" or whether you see that as distinct.